.ci/oci-e2e-build-service.sh

```diff
@@ -18,11 +18,12 @@ export APPLICATION_NAME="all-components-staging"
 # BUILD_SERVICE_IMAGE - build-service image built in openshift CI job workflow. More info about how image dependencies work in ci: https://github.com/openshift/ci-tools/blob/master/TEMPLATES.md#parameters-available-to-templates
 # Container env defined at: https://github.com/openshift/release/blob/master/ci-operator/config/redhat-appstudio/build-service/redhat-appstudio-build-service-main.yaml
 # Openshift CI generates the build service container image value as registry.build01.ci.openshift.org/ci-op-83gwcnmk/pipeline@sha256:8812e26b50b262d0cc45da7912970a205add4bd4e4ff3fed421baf3120027206. Need to get the image without sha.
-export BUILD_SERVICE_IMAGE_REPO=${BUILD_SERVICE_IMAGE%@*:-"quay.io/redhat-appstudio/build-service"}
+export BUILD_SERVICE_IMAGE=${BUILD_SERVICE_IMAGE%@*}
+export BUILD_SERVICE_IMAGE_REPO=${BUILD_SERVICE_IMAGE:-"quay.io/redhat-appstudio/build-service"}
```

Comment:
fixes [this issue](https://github.com/redhat-appstudio-qe/infra-deployments/commit/2c2c35af2450bdf43e616c6e8f9e2246e4340719#diff-5c93bf47073dc33e77c2f62d49686f096901505d3c52a8f28c7c44457b07ed8cR17) (stripping `@sha256:<commit-sha>` did not work with the old expression)